app/controllers/users_controller.rb

```diff
@@ -1,9 +1,19 @@
 class UsersController < ApplicationController
-  layout "landing"
+  layout 'landing', :only => [:new, :create]
   skip_before_action :authenticate!
 
   def index; end
 
+  def edit
+    @user = current_user
+  end
```

Comment:
use `load_and_authorize only: [:edit, :update]` so you won't need the `@user = current_user` defined in the edit.